codegen/java-gen/src/main/scala/kalix/codegen/java/EventSourcedEntitySourceGenerator.scala

```diff
@@ -88,7 +88,7 @@ object EventSourcedEntitySourceGenerator {
        |  }
        |
        |  @Override
-       |  public $stateType handleEvent($stateType state, Object event) {
+       |  public $stateType handleEvent($stateType state, Object event, Metadata metadata) {
```

Comment:
Do we need to add it to this method signature? 

Users can access it from the EventContext and the addition here is not even being used. The `eventCases` are not using this information.

Reply:
I agree, don't change this signature, it would break every existing service built with the protocol first JVM SDKs

Reply:
Ah, sorry, I thought it was the abstract event sourced entity that the user extends but it is the generated route-to-the-right-command/event-handler method.

Reply:
I'll take it from the context.

Reply:
Actually I will remove the metadata all together